CPS-XXXX/README.md

```diff
@@ -0,0 +1,156 @@
+---
+CPS: 16
+Title: Cardano URIs
+Status: Open
+Category: Tools
+Authors:
+  - Adam Dean <adam@crypto2099.io>
+Proposed Solutions:
+  - CIP-0013: https://github.com/cardano-foundation/CIPs/tree/master/CIP-0013
+  - CIP-0099: https://github.com/cardano-foundation/CIPs/tree/master/CIP-0099
+  - CIP-0107: https://github.com/cardano-foundation/CIPs/pull/635
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/issues/836
+  - https://github.com/cardano-foundation/CIPs/pull/559
+  - https://github.com/cardano-foundation/CIPs/pull/635
+  - https://github.com/cardano-foundation/CIPs/pull/841
+Created: 2024-06-15
+License: CC-BY-4.0
+---
+
+## Abstract
+
+Cardano URI schemes ([CIP-13]) have been defined since early 2021. However, some
+of the proposed standards have languished and struggled for adoption event as
+the amount of wallet developers in the ecosystem has skyrocketed. This CPS aims
+to create a centralized point of reference for those interested in creating new
+standards or publishing integration solutions.
+
+## Problem
+
+A keen pain point in the utilization of blockchain is the archaic and convoluted
+processes necessary to interact with the blockchain or cite historical ledger
+data. Well-defined URI schemes can provide a plethora of easy access and
+interaction methods, particularly to mobile users, since they can leverage
+deep-linking and QR technologies to pass data between applications.
+
+## Use cases
+
+The Cardano URI Scheme(s) that exist already present several fantastic use cases
+for URI schemes. We welcome additional or expanded definitions to be added to
+this section for further discussion, development, and adoption by the community.
+
+### Easy On-Chain Interactions
+
+#### Payments
+
+1. The original specification of [CIP-13][CIP-13-payment] defined only the
+   base `web+cardano:` **scheme** and an **authority**-less path consisting of a
+   payment address and an optional `amount` query parameter to specify via QR
+   code or deep-link an address and amount of Lovelace to send.
+2. In 2024 a new CIP[reference needed] was proposed defining an explicit `//pay`
```

Comment:
```suggestion
2. In 2024 a new CIP[reference needed] was proposed defining an explicit `//pay`
```

CIP[reference needed] Which proposal is this?

Reply:
Resolved with https://github.com/cardano-foundation/CIPs/pull/841/commits/884b6a5aeea6797b2387b2f53168cd2d572a8251.